rotkehlchen/tasks/calendar.py

```diff
@@ -365,6 +367,44 @@ def maybe_create_locked_crv_reminders(self) -> None:
             error_msg='Failed to add the CRV lock period end reminders',
         )
 
+    def maybe_create_locked_aero_vero_reminders(self) -> None:
+        """Create reminders for AERO/VERO lock periods in vote escrow history events."""
+        if len(events := self.get_history_events(
+            event_types=[
+                (HistoryEventType.DEPOSIT, HistoryEventSubType.DEPOSIT_FOR_WRAPPED),
+                (HistoryEventType.INFORMATIONAL, HistoryEventSubType.NONE),  # unlock time updates
+            ],
+            counterparties=[CPT_VELODROME, CPT_AERODROME],
+        )) == 0:
+            return
+
+        lock_entries: list[int] = []
+        for event in events:
+            if not (
+                    (extra_data := event.extra_data or {}) and
```

Comment:
```suggestion
                    (extra_data := event.extra_data) is not None and
```
If extra_data is None the other conditions in the if won't be evaluated so I don't think theres any need to make sure extra_data is set to an empty dict.